generate_postmortem.sh

```diff
@@ -817,6 +827,34 @@ for NAMESPACE in $NAMESPACE_LIST; do
         rm -fr $K8S_NAMESPACES_CONFIGMAP_DATA
     fi
 
+    #grab validataingwebhookconfiguation data 
+    OUTPUT=`kubectl get validatingwebhookconfiguration -n $NAMESPACE 2>/dev/null`
+    if [[ $? -eq 0 && ${#OUTPUT} -gt 0 ]]; then
+        echo "$OUTPUT" > "${K8S_CLUSTER_VALIDATINGWEBHOOK_CONFIGURATIONS}/validatingwebhookconfiguration.out"
+        while read line; do 
+            vwc=`echo "$line" | cut -d' ' -f1`
+            kubectl get validatingwebhookconfiguration $vwc -n $NAMESPACE -o yaml &> "${K8S_CLUSTER_VALIDATINGWEBHOOK_YAML_OUTPUT}/${vwc}.yaml"
+            [ $? -eq 0 ] || rm -f "${K8S_CLUSTER_VALIDATINGWEBHOOK_YAML_OUTPUT}/${vwc}.yaml"
+
+        done <<< "$OUTPUT" 
+    else 
+        rm -fr $K8S_CLUSTER_VALIDATINGWEBHOOK_CONFIGURATIONS
+    fi
+
+    #grab mutatingwebhookconfiguration data 
+    OUTPUT=`kubectl get mutatingwebhookconfiguration -n $NAMESPACE 2>/dev/null`
+    if [[ $? -eq 0 && ${#OUTPUT} -gt 0 ]]; then
+        echo "$OUTPUT" > "${K8S_CLUSTER_MUTATINGWEBHOOK_CONFIGURATIONS}/mutatingwebhookconfiguration.out"
+        while read line; do 
+            mwc=`echo "$line" | cut -d' ' -f1`
+            kubectl get mutatingwebhookconfiguration $mwc -n $NAMESPACE -o yaml &> "${K8S_CLUSTER_MUTATINGWEBHOOK_YAML_OUTPUT}/${mwc}.yaml"
+            [ $? -eq 0 ] || rm -f "${K8S_CLUSTER_MUTATINGWEBHOOK_YAML_OUTPUT}/${mwc}.yaml"
+
+        done <<< "$OUTPUT" 
+    else 
+        rm -fr $K8S_CLUSTER_MUTATINGWEBHOOK_CONFIGURATIONS
+    fi
+
```

Comment:
Both `validatingwebhookconfiguration` and `mutatingwebhookconfiguration` are cluster resources. No need to run the `kubectl get` command with `-n $NAMESPACE` multiple times. These code blocks should be moved to the `collect cluster specific data` part.